docs/reference/snapshot-restore/repository-s3.asciidoc

```diff
@@ -143,7 +143,9 @@ settings belong in the `elasticsearch.yml` file.
 
 `read_timeout`::
 
-    The socket timeout for connecting to S3. The value should specify the unit.
+    The maximum time {es} will wait to receive the next byte of data over an established,
```

Comment:
nit: we should really link to `time-units` here rather than explaining the syntax in words. But then if we do that we'd need to add types for all the other fields too.

```suggestion
    (<<time-units,time value>>)
    The maximum time {es} will wait to receive the next byte of data over an established,
```